### PR TITLE
Update llms card to properly use local ollama

### PR DESCRIPTION
**Pull request by golemus**
added needed environment variable and point the embedder to ollama ( othewise goes to openai)

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ X] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Please delete options that are not relevant.

- [ ] Unit Test
- [X ] Test Script (please provide)

#!/mnt/g/trabaho/embedchain/.venv/bin/python
from embedchain import App
import os

os.environ["OLLAMA_HOST"] = "http://127.0.0.1:11434"

app = App.from_config(config_path="config.yaml")

# Print the app object
#print("App object:", app)

# Print the app configuration
#print("App configuration:", app.config)

# Print the language model object
#print("Language model object:", app.llm)

app.add("https://www.forbes.com/profile/elon-musk")
app.add("https://en.wikipedia.org/wiki/Elon_Musk")
app.query("What is the net worth of Elon Musk today?")

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ X] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] Made sure Checks passed

### CODE REVIEW

File: docs/components/llms.mdx [hunk 1]
```diff
@@ -330,6 +330,7 @@ Setup Ollama using https://github.com/jmorganca/ollama
 
 ```python main.py
 import os
+os.environ["OLLAMA_HOST"] = "http://127.0.0.1:11434"
```
Comment (Dev-Khant) [inline docs/components/llms.mdx:333]: @golemus What do you need to set host here?

Comment (golemus) [reply docs/components/llms.mdx:333]: to the IP of your local ollama installation (localhost if same PC)
https://github.com/ollama/ollama/blob/main/docs/faq.md

Comment (Dev-Khant) [reply docs/components/llms.mdx:333]: Alright, got it.

**Review by deshraj — APPROVED** [review]

**Comment by deshraj** [comment]
Thanks for updating it.

**Pull request merged**